src/QMCWaveFunctions/Fermion/DiracMatrixComputeOMPTarget.hpp

```diff
@@ -0,0 +1,310 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2021 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_DIRAC_MATRIX_COMPUTE_OMPTARGET_H
```

Comment:
This is largely a copy of DiracMatrix. Is this really needed? It was only a few lines of code in the MatrixUpdateOMPTarget.h

Reply:
Needs to be determined, but I use DiracMatrixComputeXXX as a compute resource not as a "object" tat both computes and holds one walkers worth of data so that's a big issue.

Reply:
DiracMatrix is a compute handle and doesn't hold data for one walker. Is it not the case?

Reply:
The linear algebra section doesn't need to be repeated. Longer term delete DiracMatrix and the similarity of code isn't an issue.

Reply:
Since this doesn't bring any thing immediately helpful, I'm in favor of deleting this one and use the old way. Once we have the CUDA conterpart stable, we can come back to review this one and DiracMatrix. Maintaining duplicated code is not an optimal solution.

Reply:
It would also mean completely changing the CUDA implementation and returning to DiracMatrix as a data object and not just a compute object that manages the scratch space needed for its batch size of computation.